data/docs_markdown/Device_Owner_Staking.md

```
@@ -0,0 +1,47 @@
+# Device Owner Staking
+Staking **$IO tokens** is a key step in participating in the IO.net decentralized ecosystem. It helps secure the network, validate computational tasks, and earn rewards. Whether you're staking on your own or teaming up with others through **co-staking** , this guide will walk you through everything —from connecting your wallet to managing your stakes and maximizing your rewards.
+### [](https://docs.io.net/docs/device-owner-staking#table-of-contents)
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)